libcomps/docs/Doxyfile.user

```diff
@@ -1018,7 +1018,7 @@ HHC_LOCATION           =
 
 # If the GENERATE_HTMLHELP tag is set to YES, the GENERATE_CHI flag
 # controls if a separate .chi index file is generated (YES) or that
-# it should be included in the primary .chm file (NO).
+# it should be included in the master .chm file (NO).
```

Comment:
I think we can keep the "master" -> "primary" replacement...

Reply:
I agree, I added one commit to substitute master->primary